spec.bs

```diff
@@ -338,13 +338,19 @@ An aggregatable report is a [=struct=] with the following items:
 :: A [=string=]
 : <dfn>debug details</dfn>
 :: A [=debug details=]
+: <dfn>aggregation coordinator</dfn>
+:: An [=aggregation coordinator=]
 : <dfn>context ID</dfn>
 :: A [=string=] or null
 : <dfn>queued</dfn>
 :: A [=boolean=]
 
 </dl>
 
+<h3 dfn-type=dfn>Aggregation coordinator</h3>
+An aggregation coordinator is an [=origin=] that the [=allowed aggregation
```

Comment:
I added an issue here -- we don't have the 'suitable' concept in PAA yet. It might make sense to switch over, but I think currently most of the logic does not assume that the scheme is either http or https.